scripts/provisioning/s3_setup

```diff
@@ -590,10 +589,17 @@ get_platform() {
 
 if [[ $setlogrotate == true ]]
 then
-    plt=$(get_platform)
-    echo "Server type is $plt"
-    if [[ "$plt" == "virtual" ]]
-    then
-        set_logrotate_policy_ova
-    fi
+    set_logrotate_policy_ova
 fi
+
+# TODO - uncomment below section once we know how to figure out the host type (virtual or physical)
+# without using 'provisioner' cli.
+#if [[ $setlogrotate == true ]]
+#then
+#    plt=$(get_platform)
```

Comment:
yes, for now I have commented this as objective is to make this script working for single node setup. As soon as we come up with some alternative to "provisioner" cli to get type ("physical" or "virtual") of the host, this will be fixed.

We may also decide to remove this log-rotate for OVA also, asking the user to do it manually if needed.

Reply:
okay..sure